bats.yaml

```diff
@@ -28,9 +29,28 @@ pipeline:
       expected-commit: 5da66876b8b619235aee1eb3e54954eaca88059b
 
   - runs: |
-      ./install.sh  "${{targets.contextdir}}/usr"
+      ./install.sh "${{targets.contextdir}}/usr"
+      mkdir -p "${{targets.contextdir}}/tmp/"
+      mv ./docker "${{targets.contextdir}}/tmp/docker"
+      ls -la "${{targets.contextdir}}/tmp/docker/"
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" support 0.3.0
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" file 0.4.0
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" assert 2.1.0
+      "${{targets.contextdir}}/tmp/docker/install_libs.sh" detik 1.3.1
+      mkdir -p "${{targets.contextdir}}/usr/local/bin"
+      ln -s /opt/bats/bin/bats "${{targets.contextdir}}/usr/local/bin/bats"
+      find "${{targets.contextdir}}" -type f -perm /2000 -exec chmod g-s {} \;
+      rm -rf "${{targets.contextdir}}/tmp/"
 
 subpackages:
+  - name: bats-compat
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}/opt/bats/"
+          find . -maxdepth 1 -not -name '.' -not -name 'melange-out' -not -name '*.tmp' -not -name '*.log' \
+            -exec mv {} "${{targets.contextdir}}/opt/bats/" \;
```

Comment:
Is it intended that we move over everything?

Reply:
Hey RJ, thanks for the review. The reason we needed to add all of this was to match the upstream image. https://github.com/bats-core/bats-core/blob/master/Dockerfile where the entire repo is shifted and the link that I created above is also created.

Reply:
In that case, could we remove the .git folder? And anything else it may not actually be using at runtime?

Reply:
I compared the dir in my image:
```
bash-5.2$ ls /opt/bats
AUTHORS                       docker                        man
Dockerfile                    docker-compose.override.dist  package.json
LICENSE.md                    docker-compose.yml            shellcheck.sh
README.md                     docs                          test
SECURITY.md                   install.sh                    uninstall.sh
bin                           lib
contrib                       libexec
```
vs upstream
```
/code # ls /opt/bats
AUTHORS                       bin                           docs                          package.json
Dockerfile                    contrib                       install.sh                    shellcheck.sh
LICENSE.md                    docker                        lib                           test
README.md                     docker-compose.override.dist  libexec                       uninstall.sh
SECURITY.md                   docker-compose.yml            man
```
both are the same

Reply:
Should I explicitly look at removing items?

Reply:
Anushka did you address this?

Reply:
Yes Mritunjay, I have now! Thanks @kranurag7